Resources/Locale/en-US/store/uplink-catalog.ftl

```diff
@@ -14,6 +14,9 @@ uplink-rifle-mosin-desc = A bolt action service rifle that has seen many wars. N
 uplink-esword-name = Energy Sword
 uplink-esword-desc = A very dangerous energy sword that can reflect shots. Can be stored in pockets when turned off. Makes a lot of noise when used or turned on.
 
+uplink-esword-double-name = Double Bladed Energy Sword
+uplink-esword-double-desc = A much more dangerous energy sword that can reflect shots and swings twice as fast compared to its lesser counter part. Can be stored in pockets when turned off. Makes a lot of noise when used or turned on.
```

Comment:
The description should be tweaked. It's not really more dangerous than an energy sword anymore. Should also mention it has a higher chance of reflection compared to a regular energy sword and that it can also do structural damage.